main.css

```diff
@@ -667,18 +692,25 @@ a:hover .profile__portrait {
   height:         40px;
   padding:        0;
   margin-bottom:  6px;
+  margin-left:    6px;
   vertical-align: middle;
 }
 
 .github-banner {
   z-index:  1;
-  position: fixed;
+  position: absolute;
   top:      0;
   right:    0;
   width:    149px;
   height:   149px;
 }
 
+@media (min-width: 1024px) {
+  .github-banner {
+    position: fixed;
+  }
+}
```

Comment:
mobile screens are small. github banner was annoying as hell while scrolling. we only fix it on larger screens